geode-assembly/geode-assembly-test/build.gradle

```diff
@@ -21,10 +21,11 @@ apply from: "${rootDir}/${scriptDir}/standard-subproject-configuration.gradle"
 
 
 dependencies {
-  compile(platform(project(':boms:geode-all-bom')))
+  api(platform(project(':boms:geode-all-bom')))
   compileOnly(project(':extensions:geode-modules-test'))
   compileOnly(project(':geode-core'))
   compileOnly(project(':geode-pulse'))
+  implementation(project(':geode-junit'))
```

Comment:
Why is this being added here? I wouldn't expect these changes to be adding any new dependencies.

Reply:
`geode-dunit` passes `geode-junit` along as a `compile` dependency prior to this change. Modules were using `geode-junit` without explicitly asking for it. 
Alternatively, I could make `geode-dunit` declare `geode-junit` as an `api` dependency, but I think that would be lazy and technically incorrect.

Reply:
Given that some of the classes in geode-dunit extend/implement classes in good-junit it more appropriate to make it `api`.